cinema/views.py

```diff
@@ -48,6 +78,32 @@ class MovieSessionViewSet(viewsets.ModelViewSet):
     queryset = MovieSession.objects.all()
     serializer_class = MovieSessionSerializer
 
+    def get_queryset(self):
+        queryset = super().get_queryset()
+
+        date = self.request.query_params.get("date")
```

Comment:
here also - this only applicable to list action

Reply:
It's not understandable for me, why in list we should call super, but in other cases no, or what?

Reply:
@AndriiHamasa Oleksandr meant that you should handle query params only when it's list action because query params are passed only for GET(for list) http method